Sources/KlaviyoCore/Networking/SDKRequestIterator.swift

```diff
@@ -128,7 +128,6 @@ public struct SDKRequest: Identifiable, Equatable {
     }
 }
 
-@_spi(KlaviyoPrivate)
```

Comment:
Hmm does the test app not need this?

Reply:
Hmm yeah I had the same question, turns out it doesn't as long as it can access the `requestIterator`

Reply:
Ok but we aren't exposing anything in the public api that we shouldn't be right?